sklearn/preprocessing/_data.py

```diff
@@ -3266,6 +3267,18 @@ def power_transform(X, method='yeo-johnson', *, standardize=True, copy=True):
      [ 0.256... -0.707...]
      [ 1.076...  1.414...]]
 
+    .. warning:: Risk of data leak
```

Comment:
Since these lines are rendered together (https://106562-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.preprocessing.power_transform.html) we need a `.` at the end of this line.

```suggestion
    .. warning:: Risk of data leak.
```